config/initializers/country_config.rb

```diff
@@ -177,6 +97,10 @@ def self.country_environment_file
   def self.current_country?(country)
     CountryConfig.current[:name] == country
   end
+
+  def self.dhis2_data_elements
```

Comment:
Let's cache this with Memery so the file isn't loaded every time someone calls this function.